### PR TITLE
First draft of a contributing.md file

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Submitting Bug Reports and Feature Requests
+If you find a bug or can think a feature you would really like to see being implemented, you can [create a new issue](https://github.com/dh-tech/hackathon-2022/issues). Please first look through the existing issues, however, to avoid duplication of issues.
+
+If you report a bug, please include any error messages you get and a full description of the steps to reproduce the bug. For new feature requests, please clearly describe the functionality you are looking for and, if applicable, why any existing workflow does not suffice. Please also consider, fixing bugs and implementing new features yourself and submit pull request! :) 
+
+## Getting Help
+If you need help, you can open a new issue and tag it with "Help Request". A better way, however, is to join the [DHTech Slack](https://dh-tech.github.io/join/) and ask for help there.
```
[the remark this answers is not in the document]
Comment (rlskoeser) [reply CONTRIBUTING.md:20]: I vote for discussing on dh-tech slack first when possible...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Contributing to Undate - A fuzzy date Python library
+
+
+Hey there!
+
+If you found your way here that probably means you are curious about how to contribute to this project. This is great! We are always looking for new contributors. If you can't find the information you are looking for in this document or anywhere else in the repo, please consider [opening a ticket](https://github.com/dh-tech/hackathon-2022/issues) so we know there is something we need to address.
+
+## Project Setup
+Instructions on how to set up the project locally and how to run the tests can be found [in the Readme file](https://github.com/dh-tech/hackathon-2022/blob/main/README.md).
```
Comment (rlskoeser) [inline CONTRIBUTING.md:9]: Is it possible to make this a relative link so it won't break when we rename the repo?

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdamerow) [reply CONTRIBUTING.md]: How is this (in the latest commit)?

File: CONTRIBUTING.md (hunk 2 above)
Comment (jdamerow) [reply CONTRIBUTING.md]: Relative links only seem to work for other files in the repo, not for issues, pull requests etc 😕

**Review by rlskoeser — APPROVED** [review]
looks good to me, thanks for drafting and revising this!

**Pull request merged**